docs: add caution box to cluster config command (#670)
redpanda-data/documentation: docs/cluster-administration/cluster-property-configuration.mdx

2. Copy `<filename>.yaml` to the other cluster.
3. Log in to the other cluster and import the file with the saved configuration by running `rpk cluster config import –filename <filename>.yaml`. This command applies the property settings in `<filename>.yaml` to all nodes in the cluster.

:::caution

> Redpanda does not support importing cluster specific identification, e.g. `cluster_id`, with this command.

Reviewer: ```suggestion
Redpanda does not support importing cluster-specific identification (such as `cluster_id`) with this command.
```
